api/db/migrate/20210102222850_create_plan_changelogs.rb

```diff
@@ -0,0 +1,11 @@
+class CreatePlanChangelogs < ActiveRecord::Migration[6.0]
+  def change
+    create_table :plan_changelogs do |t|
+      t.string :log, null: false
+      t.bigint :author_id, null: false
+      t.string :author, null: false
```

Comment:
We don't need this as the author name can be retrieved through the author id. (and with my comment above, we can just do `planChangelog.author.name`